index.js

```diff
@@ -15,15 +15,15 @@ type Section {
     title: String!
     description: String
     pages: [Page]
-    questionnaireId: Int!
+    questionnaire: Questionnaire
```

Comment:
I wonder whether we should actually deprecate, rather than delete, `questionnaireId` etc? That way this can be safely merged and deployed at any time without breaking the front-end